uiowa/uiowa
# [Commencement] Event and venue initial theming tasks

Resolves #7600 

# How to test
```
ddev blt ds --site commencement.uiowa.edu && ddev drush @commencement.local commencement-events && ddev drush @commencement.local uli node/76/edit
```
- Add some additional information sections and save
- Confirm that additional information collections are rendering as accordions
- Confirm that event teasers are rendering on venues as cards
- Create a session and upload a PDF document to it
- Edit an event to set it to the session you created
- Confirm that program PDF link is displaying on the event node and no other session information appears with it
- Confirm that events full nodes are rendering using LB
